test/e2e/scalenodes.go

```diff
@@ -75,11 +81,41 @@ var _ = Describe("Scale nodes", func() {
 		err = waitForScale(mss.Items[0].Name)
 		Expect(err).NotTo(HaveOccurred())
 	})
+
+	Specify("operator should maintain at least three worker replicas", func() {
+		ctx := context.Background()
+		infraId, err := clients.AROClusters.AroV1alpha1().Clusters().Get(ctx, arov1alpha1.SingletonClusterName, metav1.GetOptions{})
```

Comment:
Nit: It is not `infraId`: it is an object whchi alongside other things contains infra id.

Reply:
Good catch, I have renamed to `instance`.